app/src/message/message-breadcrumbs.component.js

```diff
@@ -64,8 +65,11 @@ export default function MessageBreadcrumbs(props) {
       <div class="message-breadcrumbs">
         <nav style="--bs-breadcrumb-divider: '>';">
           <ol class="breadcrumb">
+            <li className="breadcrumb-item">
+              <Link to="/feed" class="no-decoration">{`${t("feed_group")}`}</Link>
+            </li>
             {group && (
-              <li class="breadcrumb-item">
+              <li className="breadcrumb-item">
```

Comment:
`class` is [supported in preact](https://preactjs.com/guide/v10/differences-to-react#raw-html-attributeproperty-names).

But you can keep `className`, doesn't matter

Reply:
I'll revert this. Your link says the shorter one is normally preferred.

Reply:
This is done - in fact the whole thing is reverted because of removing the feed link from breadcrumbs.